griptape/drivers/image_generation/openai_dalle_image_generation_driver.py

```diff
@@ -48,13 +50,12 @@ class OpenAiDalleImageGenerationDriver(BaseImageGenerationDriver):
     def try_generate_image(self, prompts: list[str], negative_prompts: list[str] | None = None) -> ImageArtifact:
         prompt = ", ".join(prompts)
 
-        additional_params = {}
+        additional_params = dict[str, Any]()
```

Comment:
I think we use the alternate syntax in other places. I think it's a bit easier to read.
```python
additional_params: dict[str, Any] = {}
```